### PR TITLE
[Method References 2] LPS-176118 Convert method references to serviceTracker in class ThemeContributorTopHeadDynamicInclude

### PR DESCRIPTION
**Pull request by W2jx**
Resend, I have fixed the bug https://liferay.slack.com/archives/GLCFMMZQW/p1680222170453829.

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/4018

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:frontend

**Comment by W2jx** [comment]
ci:test:bundle

**Comment by W2jx** [comment]
ci:test:acceptance-functional

File: modules/apps/frontend-theme/frontend-theme-contributor-extender/src/main/java/com/liferay/frontend/theme/contributor/extender/internal/servlet/taglib/ThemeContributorTopHeadDynamicInclude.java [hunk 1]
```diff
@@ -69,7 +71,11 @@ public void include(
 			portalCDNURL = themeDisplay.getPortalURL();
 		}
 
-		if (_cssResourceURLs.length > 0) {
+		ResourceURLsBag resourceURLsBag = _getResourceURLsBag();
+
+		List<String> cssResourceURLs = resourceURLsBag._cssResourceURLs;
+
+		if (!cssResourceURLs.isEmpty()) {
```
Comment (dantewang) [inline modules/apps/frontend-theme/frontend-theme-contributor-extender/src/main/java/com/liferay/frontend/theme/contributor/extender/internal/servlet/taglib/ThemeContributorTopHeadDynamicInclude.java:78]: @W2jx The only change compared to the last merged one is this line, right?

**Review by dantewang — APPROVED** [review]

File: modules/apps/frontend-theme/frontend-theme-contributor-extender/src/main/java/com/liferay/frontend/theme/contributor/extender/internal/servlet/taglib/ThemeContributorTopHeadDynamicInclude.java (hunk 1 above)
Comment (W2jx) [reply modules/apps/frontend-theme/frontend-theme-contributor-extender/src/main/java/com/liferay/frontend/theme/contributor/extender/internal/servlet/taglib/ThemeContributorTopHeadDynamicInclude.java:78]: Yes

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by dantewang** [comment]
ci:forward:force

cc @shuyangzhou @tinatian This pull has been sent to Brian and merged before, but reverted due to a typo/bug which is fixed here: https://github.com/liferay-core-infra/liferay-portal/pull/4101#pullrequestreview-1370279823

**Pull request closed without merging**